tools/fastlane-plugin/lib/fastlane/plugin/bugsnag/actions/send_build_to_bugsnag.rb

```diff
@@ -167,7 +170,12 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :endpoint,
                                        description: "Bugsnag deployment endpoint",
                                        optional: true,
-                                       default_value: "https://build.bugsnag.com")
+                                       default_value: "https://build.bugsnag.com"),
+          FastlaneCore::ConfigItem.new(key: :metadata,
+                                       description: "Metadata",
+                                       optional:true,
+                                       type: Object, 
+                                       default_value: nil)
```

Comment:
Added so that I could test metadata as other PR not merged yet. Will remove before merging to avoid clashes.

Reply:
In general, if your branch in dependent on another there is a trick you can pull with GitHub (basically just chain PRs together).

E.g. if `another-branch` is being PRed into `next` and you rely on `another-branch`, then you can just branch off of `another-branch` and PR into that instead.  When it's merged and the branch is deleted, your branch will magically target `next` instead.

It's a useful trick, but be a little wary with it.... if `another-branch` is changed substantially during review then you can end up in a bit of a mess.

Aside from that, note that Git is typically very tolerant of seeing the same code being added in separate branches.  If it's exactly then same then it will just come out in the wash by updating your branch (merging `next` into it), without creating conflicts.